.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,24 +16,19 @@ Related ticket(s):
 - [ ] Waited for all CI jobs to pass before requesting a review.
 - [ ] Added/updated tests and documentation where relevant.
 
-## For Reviewer/Developer
+## Merging a feature branch?
 
```

Comment:
The branch in heading can also be `chore/` `fix/`

Reply:
I've gone with "Merging a feature or bug fix?", as I don't think it makes sense to be too wordy here.